src/main/java/org/opentripplanner/inspector/NoThruTrafficEdgeRenderer.java

```diff
@@ -14,37 +14,35 @@
  */
 public class NoThruTrafficEdgeRenderer implements EdgeVertexRenderer {
 
-    private static final Color THRU_TRAFFIC_COLOR = Color.LIGHT_GRAY;
-
-    private static final Color NO_THRU_TRAFFIC_COLOR = Color.RED;
-
-    private static final Color NO_BICYCLE_THRU_TRAFFIC_COLOR = Color.BLUE;
-
-    private static final Color NO_CAR_THRU_TRAFFIC_COLOR = Color.ORANGE;
-
     public NoThruTrafficEdgeRenderer() {
     }
 
     @Override
     public boolean renderEdge(Edge e, EdgeVisualAttributes attrs) {
         if (e instanceof StreetEdge) {
             StreetEdge pse = (StreetEdge) e;
-            if (pse.isBicycleNoThruTraffic() && pse.isMotorVehicleNoThruTraffic()) {
-                attrs.color = NO_THRU_TRAFFIC_COLOR;
-                attrs.label = "no thru traffic";
-            }
-            else if (pse.isBicycleNoThruTraffic()) {
-                attrs.color = NO_BICYCLE_THRU_TRAFFIC_COLOR;
-                attrs.label = "no bicycle thru traffic";
-            }
-            else if (pse.isMotorVehicleNoThruTraffic()) {
-                attrs.color = NO_CAR_THRU_TRAFFIC_COLOR;
-                attrs.label = "no car thru traffic";
-            }
-            else {
-                attrs.color = THRU_TRAFFIC_COLOR;
-                attrs.label = "";
+	    int r = 200, g = 0, b = 0;
```

Comment:
Could you explain why removed the pre-defined colours and used the custom ones?

Reply:
Sure ... previously there was 4 combinations, now with 3 traverse modes, there are 8 of them. That would be quite an if-else maze. 

Of course there is a bit of laziness involved :) Colors could be defined in an array, indexed with a combination of traverse mode specific bits.

Reply:
Actually, no that's fine. I get it now.

So does each of the 8 combinations have its own colour?

Reply:
Yes.

Reply:
I modified the code to use a lookup table for colors coding Now it is easy to change the colors.